.gitlab-ci.yml

```diff
@@ -1,27 +1,13 @@
-image: govcmsdev/gitlab-ci
-services:
-- docker:dind
-stages:
-- build
-- deploy
-build:
-  stage: build
-  script:
-    - "export DOCKER_HOST=tcp://localhost:2375"
-    - "docker network prune -f && docker network create amazeeio-network"
-    - "docker login -u gitlab-ci-token -p $CI_JOB_TOKEN gitlab-registry-production.govcms.amazee.io"
-    - "ahoy up"
-    - "docker-compose exec -T test dockerize -wait tcp://mariadb:3306 -timeout 1m"
-    - |
-      if grep --quiet '^MARIADB_DATA_IMAGE' .env; then
-        ahoy govcms-deploy
-      else
-        ahoy -v composer install
-        ahoy -v install
-      fi
-    - ahoy lint || true
-deploy:
-  stage: deploy
-  script:
-    - "echo \"curl -X POST -d projectName=$CI_PROJECT_NAME -d branchName=$CI_COMMIT_REF_NAME http://rest2tasks.lagoon.svc:3000/deploy\""
-    - "curl -X POST -d projectName=$CI_PROJECT_NAME -d branchName=$CI_COMMIT_REF_NAME http://rest2tasks.lagoon.svc:3000/deploy"
+# This file is locked for GovCMS SaaS.
+include:
+
+  # Local overrides and defaults.
```

Comment:
what does `Local` mean? Per-project overrides maybe?

Reply:
It will be overridden and pushed, but it would also be possible to override it ways that would not be possible to push (they won't get past a vetting process which is some follow up work). This is definitely tidy up work near the end.

Reply:
Sorry, but you did not answer my question: this comment does not explain what this file is used for.

Reply:
Acknowledged that it needs a better comment.